vpn/src/main/java/com/duckduckgo/mobile/android/vpn/ui/onboarding/DeviceShieldOnboardingStore.kt

```diff
@@ -26,6 +26,9 @@ interface DeviceShieldOnboardingStore {
     fun onboardingDidShow()
     fun onboardingDidNotShow()
     fun didShowOnboarding(): Boolean
+    fun onFeatureEnabled()
+    fun onFeatureDisabled()
+    fun isFeatureEnabled(): Boolean
```

Comment:
I would rename these methods to `disableVpnFeature()`, `enableVpnFfeature()` and `isVpnFeatureEnabled()` resp.